rosmon_core/src/ui.h

```diff
@@ -37,10 +37,20 @@ class UI
 		Terminal::Parser parser;
 	};
 
+	enum FilterMode
```

Comment:
Is there a reason why this is a bitmask? Can a node be both InclusiveFiltered and ExclusiveFiltered?
Otherwise, please change it to a usual enum.

Reply:
Removed entirely, so no longer an issue.